tests/Integration/Internal/Framework/Module/Command/ActivateConfiguredModulesCommandTest.php

```diff
@@ -134,15 +133,4 @@ private function prepareTestModuleConfigurations(bool $isConfigured, int $shopId
         $shopConfigurationDao = $this->get(ShopConfigurationDaoInterface::class);
         $shopConfigurationDao->save($shopConfiguration, $shopId);
     }
-
-    private function executeCommand(array $input): void
-    {
-        $app = $this->getApplication();
-
-        $this->execute(
-            $app,
-            $this->get('oxid_esales.console.commands_provider.services_commands_provider'),
-            new ArrayInput($input)
-        );
-    }
```

Comment:
Is this change related to the Pull-Request?

Reply:
Yes, I wouldn't be able to use that function in the unit test otherwise.